core/profiles/standard/standard.install

```diff
@@ -598,17 +191,22 @@ function standard_install() {
   // Update the menu router information.
   menu_rebuild();
 
-  // Enable the admin theme.
-  db_update('system')
-    ->fields(array('status' => 1))
-    ->condition('type', 'theme')
-    ->condition('name', 'seven')
-    ->execute();
-  config_set('system.core', 'admin_theme', 'seven');
-  config_set('system.core', 'node_admin_theme', '1');
+  // Enable default and admin themes.
+  // @see system_install()
+  $default_theme = 'basis';
+  $admin_theme = 'seven';
+  config('system.core')
```

Comment:
It would be better to save the value returned from `config('system.core')` in a variable, instead of calling `config()` all the time that configuration object is necessary.